index.html

```diff
@@ -22,41 +22,51 @@ <h4>About me</h4>
-
-                Prism godard polaroid, everyday carry man braid blog organic iPhone echo park plaid kombucha microdosing. Food truck freegan wolf man braid tilde roof party edison bulb tbh. Lumbersexual art party sartorial retro. Echo park messenger bag vinyl tbh tousled, synth ethical ugh DIY la croix hell of. Wolf crucifix kitsch synth la croix ennui succulents ethical skateboard hexagon plaid whatever pork belly franzen you probably haven't heard of them.
-
-                Snackwave everyday carry scenester chartreuse narwhal live-edge VHS viral pitchfork hell of raclette chicharrones hella. Biodiesel XOXO salvia tumblr, authentic dreamcatcher tumeric paleo godard keffiyeh. Distillery messenger bag gochujang vinyl. Art party mumblecore activated charcoal tumeric, succulents food truck chia. Meh VHS cold-pressed pabst bicycle rights semiotics intelligentsia twee cronut subway tile jianbing. Narwhal copper mug quinoa mlkshk meh mustache cornhole jean shorts hoodie umami.
-
-                Meggings subway tile salvia, kinfolk bushwick seitan glossier neutra tumeric man bun austin normcore wayfarers la croix. Yr seitan twee 3 wolf moon microdosing meditation. Vegan craft beer pour-over, gastropub gochujang keffiyeh neutra tbh. Occupy keffiyeh fingerstache mumblecore, franzen small batch post-ironic art party plaid chicharrones etsy photo booth direct trade. Pork belly cloud bread banh mi sartorial trust fund pok pok.
-
-                Hot chicken banjo before they sold out gochujang flexitarian photo booth bitters pug, pork belly jean shorts raw denim. Man braid 3 wolf moon mumblecore cornhole, next level poke humblebrag banjo intelligentsia fashion axe put a bird on it yuccie celiac selfies. Af photo booth meggings freegan chicharrones typewriter. Bushwick lyft actually hexagon, squid leggings kitsch intelligentsia brunch mixtape mumblecore thundercats chartreuse.
-
-                Oh. You need a little dummy text for your mockup? How quaint.
-
-                I bet you’re still using Bootstrap too…
-            </h2>
+            <h2>PORTFOLIO</h2>
+            <h3>Scroll down</h3>
+            <img class='scrollingMouseImage pulsingImage' src='img/mousescrollitem.png' alt='Scroll-down icon'>
+            <a href='../w1_pilotShopRebuild/index.html' target='_blank'>
+                <article>
+                    <h3>PILOT SHOP</h3>
+                    <img src='img/pilotShopScreenshot.jpg' alt='A screenshot of my pilot shop rebuild'>
+                    <p>As an exercise in responsive web design we recreated the Pilot Shop digital storefront with HTML and CSS.</p>
+                </article>
+            </a>
+            <a href='' target='_blank'>
+                <article>
+                    <h3>PLACEHOLDER PROJECT</h3>
+                    <img src='img/portfolioPlaceholder1.jpg' alt='A screenshot of my pilot shop rebuild'>
```

Comment:
Maybe worth changing alt text to say more content coming soon?

Reply:
Thanks, fixed